test/extensions/tracers/skywalking/skywalking_test_helper.h

```diff
@@ -11,6 +11,7 @@ namespace Envoy {
 namespace Extensions {
 namespace Tracers {
 namespace SkyWalking {
+namespace {
```

Comment:
no anonymous namespace in header.

Reply:
OK... Although to be honest, I am a little confused.